pkgs/development/interpreters/python/passthrufun.nix

```diff
@@ -139,8 +139,16 @@ rec {
     sitePackages
     ;
   inherit sourceVersion;
-  pythonAtLeast = lib.versionAtLeast pythonVersion;
-  pythonOlder = lib.versionOlder pythonVersion;
```

Comment:
I wouldn't be opposed to just fix the underlying issue.

```
pythonAtLeast = with sourceVersion; lib.versionAtLeast "${major}.${minor}.${patch}";
pythonAtLeast = with sourceVersion; lib.versionOlder "${major}.${minor}.${patch}";

Reply:
Actually this won't work for PyPy, which is at `sourceVersion` 7.3.17.